examples/transactions-distributed/tests/xa_transactions_test.bal

```diff
@@ -0,0 +1,28 @@
+import ballerina/test;
+import ballerina/io;
+
+any[] outputs = [];
+int count = 0;
+
+@test:Mock {
+    moduleName: "ballerina/io",
+    functionName: "println"
+}
+public function mockPrint(any... s) {
+    outputs[count] = string.convert(s[0]);
+    count += 1;
+}
+
+@test:Config
+function testFunc() {
```

Comment:
I can't see this output in ballerina-lang/examples/transactions-distributed/participant.out file. Is this correct?

Reply:
This file was mistakenly committed, we were looking at a way to test distributed scenario, specially the interleaving log/print entries.